docs/statsig-warehouse-native/getting-started.md

```diff
@@ -31,6 +31,19 @@ _Note - while you're here, you can go to **Basic Settings** and add any Custom I
 
 ## Step 2 - Create a Metric Source
 
+### Overview
+
+A Metric Source gives you the ability to define a custom SQL query into your Data Warehouse that generates a resultset on which you'll be able to use Statsig's Metric builder to define Metrics. Typically, customer data pipelines collecting a variety of categories of data from many data sources will result in a number of tables in the data warehouse. For example:
+
+* Web Clickstream events
+* Purchase events
+* Support Events (ie; via offline and callcenter data)
+* Product Return events
```

Comment:
Could you make the casing consistent here? I think only the first word in each line needs to be capitalized

Reply:
fixed